scripts/upload-api-specification.js

```diff
@@ -0,0 +1,56 @@
+const API_KEYS = require('./API_KEYS');
+const getSpecMetadata = require('./get-spec-metadata');
+const updateExistingSpec = require('./update-existing-spec');
+const createNewSpec = require('./create-new-spec');
+const fs   = require('fs');
+const path = require("path");
+
+require('dotenv').config();
+
+module.exports = async function uploadAPISpecification(filePath) {
+  
+  // Skip non-yaml files
+  if (filePath.slice(-5) !== '.yaml') {
+    console.log('[INFO] skipping upload');
+    return;
+  }
+  
+  const [version_number, file_name] = filePath.split('descriptions/')[1].split('/');
```

Comment:
It's probably safer to calculate the API version from the spec itself.

https://github.com/intercom/Intercom-OpenAPI/blob/0f1c2e051c5afb6d4d7d8805ffcdc610532f146b/descriptions/api.intercom.io.yaml#L5